doc/source/whatsnew/v0.25.0.rst

```diff
@@ -82,7 +82,7 @@ Other Enhancements
 - :meth:`DataFrame.query` and :meth:`DataFrame.eval` now supports quoting column names with backticks to refer to names with spaces (:issue:`6508`)
 - :func:`merge_asof` now gives a more clear error message when merge keys are categoricals that are not equal (:issue:`26136`)
 - :meth:`pandas.core.window.Rolling` supports exponential (or Poisson) window type (:issue:`21303`)
--
+- :class: `pandas.offsets.BusinessHour` supports multiple opening hours intervals
```

Comment:
No space between ``:class:`` and ``pandas.offsets.BusinessHour``. Also can you include the issue number this PR closes?

Reply:
Updated.